pygls/workspace.py

```diff
@@ -99,6 +99,7 @@ def _apply_none_change(self, change: TextDocumentContentChangeEvent) -> None:
 
         Currently does nothing, provided for consistency.
         """
+        pass
 
     def apply_change(self, change: TextDocumentContentChangeEvent) -> None:
```

Comment:
I think you should revert changes in `apply_change` method. Parameter `change` of this method is actually a _namedtuple_ in a runtime and some clients do not send `change.range`, so we will end up with `AttributeError`. See #81 and #101.